src/applications/personalization/rated-disabilities/util/index.js

```diff
@@ -0,0 +1,10 @@
+import { apiRequest } from 'platform/utilities/api';
+
+export async function getData(apiRoute, options) {
+  try {
+    const response = await apiRequest(apiRoute, options);
+    return response.data.attributes;
+  } catch (error) {
+    return error;
+  }
+}
```

Comment:
It looks like this came from the `profile360` app. I would just import the function from there, using `import { getData } from 'applications/personalization/profile360/util`. This isn't much different than just using `apiRequest` directly, so don't feel like you need to be use to that function just because the profile uses it.

Reply:
Ah okay. Wasn't sure if it'd be better to recreate a separate function, in the event the one from `applications/personalization/profile360/util` changed for some reason. But, that's what I was doing before, so I can just re-add the import.